website/blog/2022-04-25-jest-28.md

```diff
@@ -0,0 +1,153 @@
+---
+title: 'Jest 28: Shedding weight and improving compatibility 🫶'
+author: Simen Bekkhus
+authorURL: https://github.com/SimenB
+authorImageURL: https://avatars.githubusercontent.com/u/1404810
+---
+
+Jest 28 is finally here, and it comes with some long requested features such as support for [sharding](/docs/cli#--shard) a test run across multiple machines, [package `exports`](https://nodejs.org/api/packages.html#exports) and the ability to customize the behavior of [fake timers](/docs/jest-object#fake-timers). These are just some personal highlights, and we'll be highlighting more in this blog post.
+
+Additionally, as announced in the [Jest 27 blog post](/blog/2021/05/25/jest-27) last year, we have removed some packages that no longer are default from the default distribution. As a result the installation size has dropped by about 1/3.
+
+<!--truncate-->
+
+## Breaking changes
+
+The list of breaking changes is long (and can be seen fully in the [changelog](https://github.com/facebook/jest/blob/main/CHANGELOG.md#2800)), but for migration purposes, we've also written [a guide](/docs/upgrading-to-jest28) you can follow. Hopefully this makes the upgrade experience as frictionless as possible!
+
+Main breaking changes likely to impact your migration are dropped support for Node 10 and 15 (but _not_ Node 12, which will be EOL in a few days) and some renamed configuration options.
+
+Additionally, both of the removed modules (`jest-environment-jsdom` and `jest-jasmine2`) are still actively maintained and tested in the same way, so the only breaking change here is that you'll need to explicitly install them.
+
+The guide should hopefully make migration trivial, but note that if you use any of the packages Jets consists of directly, instead of just running `jest`, then you need to go through the changelog to view any breaking changes.
```

Comment:
```suggestion
The guide should hopefully make migration trivial, but note that if you use any of the packages Jest consists of directly, instead of just running `jest`, then you need to go through the changelog to view any breaking changes.
```